app/page/index.cjsx

```diff
@@ -8,12 +8,17 @@ Icon               = require '../component/icon/icon'
 Link               = require 'react-router/lib/Link'
 MapWithTracking    = require '../component/map/map-with-tracking'
 FeedbackPanel      = require '../component/feedback/feedback-panel'
+ItinerarySearchAction = require '../action/itinerary-search-action'
 
 class Page extends React.Component
   @contextTypes:
     getStore: React.PropTypes.func.isRequired
     executeAction: React.PropTypes.func.isRequired
 
+  componentDidMount: ->
+    if window.location.search?.indexOf('citybikes') > -1
```

Comment:
also, this should be moved to TopLevel -component, so that it is not page specific